src/docbuilder/rustwide_builder.rs

```diff
@@ -350,7 +350,11 @@ impl RustwideBuilder {
                     if res.result.successful {
                         if let Some(name) = res.cargo_metadata.root().library_name() {
                             let host_target = build.host_target_dir();
-                            has_docs = host_target.join("doc").join(name).is_dir();
+                            has_docs = if metadata.proc_macro {
+                                host_target.join("doc").join(name).is_dir()
+                            } else {
+                                host_target.join("doc").join(HOST_TARGET).join(name).is_dir()
```

Comment:
I think this needs to be `default_target`, in case the library does not build for the host target.